wordle-web/src/views/WordleView.vue

```diff
@@ -1,13 +1,59 @@
 <template>
-  <h1>Wordle Mind Bender</h1>
-
   <GameBoard :game="game" @letterClick="addChar" />
 
   <KeyBoard @letterClick="addChar" :guessedLetters="game.guessedLetters" />
 
+  <div></div>
+
   <v-btn @click="checkGuess" @keyup.enter="checkGuess"> Check </v-btn>
 
-  <h2>{{ guess }}</h2>
+  <div></div>
+
+  <v-btn @click="showBigDialog = !showBigDialog"
+    >Possible Guesses (with known letters): {{ game.possibleWords.length }}</v-btn
+  >
+
+  <div></div>
+
+  <v-btn @click="showDialog = !showDialog"
+    >Possible Guesses (with current letters): {{ game.possibleGuess.length }}</v-btn
+  >
+
+  <template>
+    <v-dialog v-model="showBigDialog">
+      <v-card>
+        <v-hover>
+          <div v-if="game.possibleWords.length < 300">
+            <div v-for="word in game.possibleWords" :key="word">
+              <v-btn @click=";[setGuess(word), (showBigDialog = !showBigDialog)]">{{ word }}</v-btn>
+            </div>
```

Comment:
The possible guesses button is duplicated

Reply:
Currently, I have two guess buttons with one of the guess buttons shows the number of possible words based on your previous guesses while the other one shows the number of words that can fill out your current guess by filtering the previous list.  I left both in there because I wasn't sure which one the teacher wanted, although I should probably give them more intuitive names.